github_connector/models/github_organization_serie.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class GithubOrganizationSerie(models.Model):
+    _name = 'github.organization.serie'
```

Comment:
I get this warning:
`github.organization.serie has no _description `
so I think It's better that you add the parameter
`_description`